src/components/PatientRegistration/PatientRegistration.tsx

```diff
@@ -138,9 +149,53 @@ export default function PatientRegistration(): ReactElement {
       relationshipChild: relationshipChild,
 
     };
-    console.log(data);
+
+    if (data.checkBoxesСonditions.conditions.size == 0 ) {
+      setConditionsError('Some should be chosen')
+    } else if (data.checkboxesFollowing.size == 0) {
+      setFollowingError('Some should be chosen')
+    } else if (stressfulLevel === undefined) {
+      setStressfulLevelError('Note the level of stress')
+    } else if (phone === "") {
+      setPhoneError("You must enter phone number")
+    } else if (email === "") {
+      setEmailError("You must enter phone email")
+    } else if (medications === "") {
+      setMedicationsError("You have enter the medication you are taking")
+    } else {
+      setConditionsError("");
+      setFollowingError("");
+      setStressfulLevelError("");
+      setPhoneError("");
+      setEmailError("");
+      setMedicationsError("");
+      // authApi.registrationClient(data)
```

Comment:
do you really need this commented line?

Reply:
yes ) 
this is how data is sent to the backend